tool/tsh/tsh.go

```diff
@@ -222,12 +225,17 @@ func main() {
 }
 
 const (
-	clusterEnvVar          = "TELEPORT_SITE"
-	clusterHelp            = "Specify the cluster to connect"
-	bindAddrEnvVar         = "TELEPORT_LOGIN_BIND_ADDR"
 	authEnvVar             = "TELEPORT_AUTH"
-	browserHelp            = "Set to 'none' to suppress browser opening on login"
+	clusterEnvVar          = "TELEPORT_CLUSTER"
+	loginEnvVar            = "TELEPORT_LOGIN"
+	bindAddrEnvVar         = "TELEPORT_LOGIN_BIND_ADDR"
+	proxyEnvVar            = "TELEPORT_PROXY"
+	siteEnvVar             = "TELEPORT_SITE"
```

Comment:
Add something explaining that `TELEPORT_SITE` is the old/deprecated version of `TELEPORT_CLUSTER` but that they mean the same thing.

Also consider adding a deprecation timeline for `TELEPORT_SITE` i.e. `// REMOVE IN: 7.0`